Gemfile

```diff
@@ -1,23 +1,25 @@
-source "https://rubygems.org"
+# frozen_string_literal: true
```

Comment:
Have these been through rubocop instead of chefstyle?

Reply:
@PrajaktaPurohit  Yes,these have been through rubocop.

Reply:
We need to run these through chefstyle instead to get the linter to pass.

Reply:
Yeah plain rubocop adds the frozen_string_literal comments which broke everything and why we don't have that turned on in chefstyle.

All the chef repos that aren't cookbooks need to be run through chefstyle, not rubocop (or cookstyle, which is for cookbooks-only)